src/Controller/CommandExecuteImmediateController.php

```diff
@@ -8,7 +8,7 @@
 use Symfony\Component\HttpFoundation\Response;
 use Synolia\SyliusSchedulerCommandPlugin\Service\ExecuteScheduleCommand;
 
-class ScheduledCommandExecuteImmediateController extends AbstractController
+class CommandExecuteImmediateController extends AbstractController
```

Comment:
Be careful, this change may be a Breaking Change

Reply:
Yes, what do you advise?

Reply:
After @Jibbarth 's good remark, I prefer to change the PhpMD rules so as not to introduce a BC break.